hlog/logrus/main.go

```diff
@@ -18,24 +18,52 @@ package main
 
 import (
 	"context"
+	"fmt"
+	"os"
+	"path"
+	"time"
 
 	"github.com/cloudwego/hertz/pkg/app"
 	"github.com/cloudwego/hertz/pkg/app/server"
 	"github.com/cloudwego/hertz/pkg/common/hlog"
 	"github.com/cloudwego/hertz/pkg/protocol/consts"
-	hertzlogrus "github.com/hertz-contrib/obs-opentelemetry/logging/logrus"
-	"github.com/sirupsen/logrus"
+	hertzlogrus "github.com/hertz-contrib/logger/logrus"
+	"gopkg.in/natefinch/lumberjack.v2"
 )
 
 func main() {
 	h := server.Default()
 
-	// For logrus detailed settings, please refer to https://github.com/hertz-contrib/obs-opentelemetry/tree/main/logging/logrus and https://github.com/sirupsen/logrus
-	logger := hertzlogrus.NewLogger(
-		hertzlogrus.WithTraceHookErrorSpanLevel(logrus.WarnLevel),
-		hertzlogrus.WithTraceHookLevels(logrus.AllLevels),
-		hertzlogrus.WithRecordStackTraceInSpan(true),
-	)
+	// Customizable output directory.
+	var logFilePath string
+	dir := "./hlog"
+	logFilePath = dir + "/logs/"
+	if err := os.MkdirAll(logFilePath, 0o777); err != nil {
+		fmt.Println(err.Error())
```

Comment:
May return here? If the error happens here, there is no meaning for the rest of the DEMO?

Reply:
Addendum: It is not recommended to use `fmt.Println` to print errors, we should use logging.

Reply:
done